v1/src/main/java/com/google/cloud/teleport/bigtable/AvroToBigtable.java

```diff
@@ -109,8 +109,7 @@ public interface Options extends PipelineOptions {
     @TemplateParameter.GcsReadFile(
         order = 5,
         description = "Input Cloud Storage File(s)",
-        helpText = "The Cloud Storage location of the files you'd like to process.",
-        example = "gs://your-bucket/your-files/*.avro")
+        helpText = "The Cloud Storage path pattern where data is located. For example, `gs://mybucket/somefolder/prefix*`.")
```

Comment:
I think leaving this as an example is fine (and is helpful for the UI) - it will get auto-generated anyways, lets not worry about formatting these - https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/499954cc43c5d767e4a649bd61d44d19b1b246ec/v1/README_GCS_Avro_to_Cloud_Bigtable.md?plain=1#L24